cv_next/server/api/users.ts

```diff
@@ -97,24 +95,24 @@ export default class UsersApi {
   }
 
   private static async getAllUsersByQueryFilter(
-    w?: QueryFieldFilterConstraint,
+    w?: Filter,
     filterOutInactive: boolean = true
   ): Promise<UserModel[] | null> {
     try {
-      let collectionRef = collection(
-        FirebaseHelper.getFirestoreInstance(),
+      let collectionRef = FirebaseHelper.getFirestoreInstance().collection(
         UserModel.CollectionName
       );
+      
       let querySnapshot: QuerySnapshot;
-      let activeWhere = where("active", "==", true);
+      let activeWhere = collectionRef.where("active", "==", true);
       if (w !== undefined) {
         const q = filterOutInactive
-          ? query(collectionRef, w, activeWhere)
-          : query(collectionRef, w);
+          ? collectionRef.where(w).where(activeWhere)
```

Comment:
Can the new package allow for editing the query after initialization?
If so, a better code will be:
`let q = collectionRef.where(w);
if (filterOutInactive) {
    q = q.wheree(activeWhere); //or something similar
}`

Reply:
Also as `filterOutInactive` check is probably common, if using this you can refactor to a separate function that'd be helpful.